extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/util/PropertyUtils.java

```diff
@@ -41,8 +46,13 @@ public static State getState(ChannelUID channelUID, AstroChannelConfig config, O
         if (value == null) {
             return UnDefType.UNDEF;
         } else if (value instanceof Calendar) {
-            Calendar cal = (Calendar) value;
-            return new DateTimeType(DateTimeUtils.applyConfig(cal, config));
+            // TODO use LocationProvider service once it's merged https://github.com/eclipse/smarthome/issues/3936
+            ZoneId zone= ZoneId.of("+03:00");
```

Comment:
This seems to be a rather arbitrary value. Is the timezone supposed to be "+3" always?